packages/devtools/lib/src/main.dart

```diff
@@ -132,71 +130,94 @@ class PerfToolFramework extends Framework {
 
   Future<bool> isFlutterApp() async {
     final EvalOnDartLibrary flutterLibrary = EvalOnDartLibrary(
-      flutterLibraryUriCandidates,
+      [flutterLibraryUri, flutterWebLibraryUri],
       serviceManager.service,
     );
 
     try {
-      await flutterLibrary.libraryRef.catchError(
-        (_) => throw FlutterLibraryNotFound(),
-        test: (e) => e is LibraryNotFound,
-      );
-    } catch (e) {
-      if (e is FlutterLibraryNotFound) {
-        return false;
-      }
+      await flutterLibrary.libraryRef;
+    } on LibraryNotFound catch (_) {
+      return false;
+    }
+    return true;
+  }
+
+  Future<bool> isFlutterWebApp() async {
+    // TODO(kenzie): fix this if screens should still be disabled when flutter
+    // merges with flutter_web.
```

Comment:
File a bug to track this.

Reply:
done - https://github.com/flutter/devtools/issues/466.